source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -56,6 +56,7 @@
         }
         this.save = function () {
             var updateCalificators = this.calificatorsManagerModel.toJs();
+            var updateEvaluators = this.calificatorsManagerModel.toEvaluators();
```

Comment:
I am not sure about the name `toEvaluators`, I mean this method extract evaluator names, right?